packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -41,6 +41,16 @@ class SKPaymentQueueWrapper {
   SKPaymentQueueDelegateWrapper? _paymentQueueDelegate;
   SKTransactionObserverWrapper? _observer;
 
+  /// Calls [`[SKPaymentQueue storefront]`](https://developer.apple.com/documentation/storekit/skpaymentqueue/3182430-storefront?language=objc)
```

Comment:
Let's document that it just returns `nil` (vs throwing) on versions where it's not available.